api/payment/stripeRouter.js

```diff
@@ -0,0 +1,39 @@
+//IMPLEMENTATION:
+//npm i stripe   ***not "stripe-checkout" but "stripe"***
+//import router into index
+//server router
+//.env file --> "STRIPE_KEY=" --> stripe test account public key
+
+// Code only need uuid installed??( line 15) and to be uncommented
+
+const express = require('express');
+const stripe = require('stripe')(process.env.STRIPE_KEY);
+const uuid = require('uuid');
+const router = express.Router();
+
+router.post('/payment', (req, res) => {
+  const { product, token } = req.body;
+  const idempotencyKey = uuid();
+
```

Comment:
router.post('/payment', (req,res) => {
const body ={ 
source: req.body.token.id,
amount: req.body.amount, //call amount when we generate request on front end
currency: 'USD'
}
})